executor/executor.go

```diff
@@ -21,7 +23,10 @@ func (e ErrStatus) Error() string {
 }
 
 // Run executes a slice of CommandDefs
-func Run(output io.Writer, job screwdriver.JobDef) error {
+func Run(path string, output io.Writer, job screwdriver.JobDef) error {
+	if err := chdir(path); err != nil {
```

Comment:
Good point. Seems like setting the dir for each command would be more
reliable, especially since commands could change the working directory.
Will change :)

On Aug 20, 2016 8:21 PM, "Jeremiah Wuenschel" <notifications@github.com>
wrote:
>
> In executor/executor.go:
>
> > @@ -21,7 +23,10 @@ func (e ErrStatus) Error() string { > } > > // Run
executes a slice of CommandDefs > -func Run(output io.Writer, job
screwdriver.JobDef) error { > +func Run(path string, output io.Writer, job
screwdriver.JobDef) error { > + if err := chdir(path); err != nil {
>
> Why are we doing this instead of just passing it as the working directory
for execute?(os/exec.cmd.Dir)
>
> Like this:
https://github.com/screwdriver-cd/launcher/blob/master/git/git.go#L111
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.